# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../licq/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur licq-1.3.2-orig/include/licq_buffer.h licq-1.3.2/include/licq_buffer.h
--- licq-1.3.2-orig/include/licq_buffer.h	2004-02-04 17:00:20.000000000 +0000
+++ licq-1.3.2/include/licq_buffer.h	2006-04-03 18:53:19.368716250 +0000
@@ -62,7 +62,7 @@
 
    char *Pack(CBuffer *);
    char *Pack(const char *data, int size);
-   char *CBuffer::PackLNTS(const char *);
+   char *PackLNTS(const char *);
    char *PackString(const char *data, unsigned short max = 0);
    char *PackUnsignedShort(unsigned short data);
    char *PackUnsignedLong(unsigned long data);
